lib/suggest.js

```diff
@@ -78,7 +79,6 @@ function suggest(value) {
   while (++index < length) {
     character = value.charAt(index)
     insensitive = character.toLowerCase()
-    upper = insensitive !== character
```

Comment:
We don't need to know the case of `character`, as we will be making our replacements at different indices.